### ISSUE TITLE
Launchpad Specific Section

### ISSUE THREAD

**Issue by kt-wawro** [ipfs-shipyard/ecosystem-dashboard#402]
Hi, 
I'm Katie a new TPM for Launchpad.

In order to measure engagement during and after launchpad, I was wondering if we would be able to make a Launchpad-specific section under orgs. Ideally, this would be everyone who has gone through the program. I'm not 100% sure if this is possible or if I would have to create a separate dashboard but figured I would ask.

**Comment by andrew** [comment]
Yeah I think this would be possible, we might need to put together a manual list of projects to display there unless there's some way this can be discovered automatically?
